WTWSMS/events/religious_events.txt

```diff
@@ -191,6 +199,7 @@ character_event = {
 		OR = {
 			religion = catholic
 			religion = fraticelli
+			religion = nicene
```

Comment:
That wasn't a matter of ordering you to do something. The religious head is suggesting/leveraging his spiritual position to get you to do something he wants. It's not an order.